extensions/blocks/rating-star/rating-meta.php

```diff
@@ -61,6 +61,26 @@ function jetpack_rating_meta_get_symbol_high_fidelity( $attributes, $pos ) {
 	}
 }
 
+if ( ! function_exists( 'jetpack_rating_get_schema_for_symbol' ) ) {
+	/**
+	 * Returns an itemprop and content for rating symbols
+	 *
+	 * @param  integer $position   the position of the symbol.
+	 * @param  integer $max_rating the maximum symbol score.
+	 *
+	 * @return string
+	 */
+	function jetpack_rating_get_schema_for_symbol( $position, $max_rating ) {
+		$schema = '';
+		if ( 1 === $position ) {
+			$schema = 'itemprop="worstRating" content="1"';
```

Comment:
Should it be `0.5` since you can assign half stars?

Reply:
Good catch, that could well be needed. I will do some testing

Reply:
Pushed a fix for this